qpc/utils.py

```diff
@@ -54,7 +54,7 @@
 
 LOG_LEVEL_INFO = 0
 
-QPC_MIN_SERVER_VERSION = "1.14.0"
+QPC_MIN_SERVER_VERSION = "2.0.0"
```

Comment:
This is what causes all Jenkins tests to fail. It always happens during these upgrades. We should ignore Jenkins job results here and merge based on standalone job results.